src/main/java/uk/gov/register/presentation/ListValue.java

```diff
@@ -21,7 +22,7 @@ public boolean isList() {
     @Override
     @JsonIgnore
     public String getValue() {
-        throw new UnsupportedOperationException();
+        return "[ " + String.join(", ", Lists.transform(elements, FieldValue::getValue)) + " ]";
```

Comment:
I agree, this object should not render the html. I see getValue() method kind of toString() method for this object. 
It does not renders linkValues as links and we need more work to do that (stated in commit message). This is quick solution to resolve 500 response from register register.